packages/blade/src/components/Button/Button.native.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement } from 'react';
+import { TouchableOpacity, Text, StyleSheet } from 'react-native';
+import { Button as ButtonProps } from './Button';
```

Comment:
```suggestion
import { Button as ButtonProps } from './Button.d';
```

Reply:
it is resolving the `d.ts` directly. why do you want me to explicitly add it?

Reply:
Since you've done that on `Button.web.tsx`